ante/sendblock.go

```diff
@@ -70,9 +73,14 @@ func (decorator *SendBlockDecorator) CheckIfBlocked(msgs []sdk.Msg) error {
 		return nil
 	}
 	for _, msg := range msgs {
-		signers := msg.GetSigners()
+		// UNFORKING v2 TODO: GetSigners is no longer available
```

Comment:
.

Reply:
Im now tracking in the description of this PR what will need to be tested:

> Check on ante CheckIfBlocked, smartaccount AnteHandle, smartaccount ValidateAuthenticatorFeePayer, smartacount GenerateAuthenticationRequest, smartaccount PostHandle. These all use GetMsgV1Signers to determine the signers, since GetSigners() was removed.

Reply:
I think the tests covering this are enough to know it works. We can do one manual test to be sure, but at least on the smart accounts tests we can see that the new way of getting the signers works properly.